server/controllers/games-controllers.js

```diff
@@ -9,3 +12,17 @@ exports.getCategories = (req, res, next) => {
       console.log("error").next(err);
     });
 };
+
+exports.getreviewById = (req, res, next) => {
+  const { review_id } = req.params;
+  selectReviewById(review_id)
+    .then((review) => {
+      if (review === undefined) {
```

Comment:
At the moment, you have error handling in both your model and controllers. Error handling should be consistent and in a single place. I would suggest the model as that is where you have rejected your promises.